src/content/migrate/4.md

```diff
@@ -37,7 +44,9 @@ module.exports = {
 }
 ```
 
-Alternatively you can pass it via CLI: `--mode production`/`--mode development`
+Alternatively you can pass it via CLI: `--mode production`/ `--mode development`
+
+T> Since the goals of the `development` and `production` versions are different. You could use `webpack-merge` to use the best of both modes. Read more [here](https://github.com/webpack/webpack.js.org/blob/master/src/content/guides/production.md)
```

Comment:
I dont think that this is something to be added to a migration guide. We already have webpack-merge mentioned in the Production guide too, maybe there  is a better place to talk about it? By the way did that link work for you locally?

Reply:
Maybe it could serve as heads up, that a certain feature is available. But I can remove if you think it doesn't need to be there. The link works but I've corrected it now.